components/camel-kafka/src/main/java/org/apache/camel/component/kafka/consumer/support/ProcessingResult.java

```diff
@@ -21,14 +21,16 @@
 
 public final class ProcessingResult {
     private static final ProcessingResult UNPROCESSED_RESULT
-            = new ProcessingResult(false, AbstractCommitManager.START_OFFSET, false);
+            = new ProcessingResult(false, -1, AbstractCommitManager.START_OFFSET, false);
```

Comment:
Please, use a constant here and document it. That way it may be easier for us to understand the code later on.